shanoir-ng-studies/src/main/java/org/shanoir/ng/study/service/RabbitMQStudiesService.java

```diff
@@ -75,17 +75,68 @@
 				return Collections.emptyList();
 			}
 			// Filter administrators and map to get only IDs
-			List<Long> ids = study.getStudyUserList().stream()
+			return study.getStudyUserList().stream()
 					.filter(studyUser -> studyUser.isReceiveNewImportReport())
```

Comment:
Hi Jean-Come, why is here the flag receiveNewImportReport checked and not the right can_administrate? Maybe I am overseeing something. With kind regards, Michael

Reply:
I just corrected a sonarlint warning here stating that we should not create a new variable "ids" just before returning it.

This method is used in EmailServiceImpl.getStudyAdministrator to get the list of users of the study that needs to receive the import mail. I think the method is just really really badly named.